editor/src/clj/editor/sprite.clj

```diff
@@ -99,7 +99,7 @@
                         vertex-data (if (= :size-mode-auto size-mode)
                                       (texture-set/vertex-data animation-frame)
                                       (slice9/vertex-data animation-frame size slice9 :pivot-center))]
-                    (assoc vertex-data :page-index (:page-index animation-frame))))
+                    (assoc vertex-data :page-index (:page-index animation-frame 0))))
```

Comment:
Unrelated fix: Avoid putting in a `nil` `:page-index` when there is no `animation-frame`.